data/card_catalogue/upgradeable-1.json

```diff
@@ -395,67 +235,30 @@
     "tier": "gold",
     "details": {
       "rules": "All players will critically succeed on their next d20 roll this turn.",
-      "flavour": "And the light shone bright upon the chosen.",
+      "flavour": "The light will shine bright upon the chosen.",
       "image": "assets/core/card/illustration/Wish.png"
     }
   },
-  {
-    "title": "Distract",
-    "upgrade": "Lie",
-    "tier": "blue",
-    "details": {
-      "rules": "Target NPC fails their current action and loses focus on you",
-      "flavour": "\"Ok one sec my homework is in my bag I'll grab it in a minute. How was your weekend?\" - Little Sniver",
-      "image": "assets/core/card/illustration/Distract.png"
-    }
-  },
-  {
-    "title": "Lie",
-    "upgrade": "Disguise",
-    "tier": "green",
-    "details": {
-      "rules": "Target NPC will believe your next statement.",
-      "flavour": "\"Yeah of course I did my homework\" - Little Sniver",
-      "image": "assets/core/card/illustration/Lie.png"
-    }
-  },
-  {
-    "title": "Disguise",
-    "upgrade": "Revise",
-    "tier": "red",
-    "details": {
-      "rules": "Change your appearance to match that of another exactly for as long as you'd like.",
-      "flavour": "\"I'm Revins, Sniver went into the other room to get his homework.\" - Little Sniver",
-      "image": "assets/core/card/illustration/Disguise.png"
-    }
-  },
-  {
-    "title": "Revise",
-    "upgrade": "",
-    "tier": "gold",
-    "details": {
-      "rules": "Rewrite the contents of a visible character's memory",
-      "flavour": "\"I'm at the end of my rope. I am done with homework. I will simply erase homework from the memory of the people\" - Little Sniver",
-      "image": "assets/core/card/illustration/Revise.png"
-    }
-  },
   {
     "title": "Cure",
-    "upgrade": "Restore",
+    "upgrade": "Field Medicine",
     "tier": "blue",
     "details": {
-      "rules": "Heal 8 to a visible character",
-      "flavour": "\"Yeah that feels a little better\" - Hugo, Recently Decapitated",
+      "rules": "Cure the ailment of a visible character, or heal them 8.",
```

Comment:
I added the cure ailment cause as a gag for when everybody is full HP. You could heal an enemy NPCs stomach ache or something to get on their good side